packages/flutter_driver/lib/src/common/diagnostics_tree.dart

```diff
@@ -34,10 +34,10 @@ class GetDiagnosticsTree extends CommandWithTarget {
 
   /// Deserializes this command from the value generated by [serialize].
   GetDiagnosticsTree.deserialize(Map<String, dynamic> json)
```

Comment:
Maybe we should change the signature here to only accept Map<String, String>? That would also be in line with what the doc comment says (serialize returns Map<String, String>).

Reply:
Or the opposite (make `serialize` return `Map<String, dynamic>`)

The possible types returned by `json.decode` in dart are : `num`, `bool`, `String`, `null`, `List<dynamic>` or a `Map<String, dynamic>`

One can find strange to name a var/param `json` without having one of the above type.

Reply:
I believe the mechanism we use to transfer this commands only supports Map<String, String>. So, calling this parameter "json" was probably slightly misleading.

Reply:
type updated